org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/XMLFormattingOptions.java

```diff
@@ -45,6 +46,7 @@
 	private static final String PRESERVE_ATTR_LINE_BREAKS = "preserveAttributeLineBreaks";
 	private static final String PRESERVE_EMPTY_CONTENT = "preserveEmptyContent";
 	private static final String SPLIT_ATTRIBUTES_INDENT_SIZE = "splitAttributesIndentSize";
+	private static final String CLOSING_TAG_NEW_LINE = "closingTagNewLine";
```

Comment:
In `<aaa></aaa>`, `<aaa>` is the opening tag, and `</aaa>` is the closing tag. Since this setting is just to put the first `>` on a new line, I think we will need to come up with a better name for the setting. Maybe `closingAngleBracketNewLine`?

Reply:
Would `closingBracketNewLine` suffice?